src/main/groovy/wooga/gradle/appcenter/api/AppCenterRest.groovy

```diff
@@ -318,6 +327,8 @@ class AppCenterRest {
                             sleep(1000)
                             break
                     }
```

Comment:
We need a break after the 200 check. See the failing tests in Jenkins.

Reply:
Got it.